scheduler-interface-spec.md

```diff
@@ -774,19 +774,14 @@ message UpdateContainerSchedulingStateRequest {
 }
 
 message UpdateConfigurationRequest {
-    // New config what needs to be saved
-    string configs = 1;
-}
-
-message UpdateConfigurationResponse {
-    // flag that marks the config update success or failure
-    bool success = 1;
+    // RM ID to update
+    string rmID = 1;
 
-    // the old configuration what was changed
-    string oldConfig = 2;
+    // PolicyGroup to update
+    string policyGroup = 2;
 
-    // reason in case of failure
-    string reason = 3;
+    // New configuration to update
+    bytes config = 3;
```

Comment:
Can we do this backward compatible and use `reserved` field number and name to indicate that one has been used. Start the new field count from 2.
I know this is overhead but lets keep it clean:
```
  reserved 1;
  reserved "configs";
```

Reply:
Updated.